dashboard/app/controllers/home_controller.rb

```diff
@@ -34,6 +34,23 @@ def health_check
 
   GALLERY_PER_PAGE = 5
   def index
+    if request.cookies['pm'] == 'new_header'
+      redirect_to '/home'
+    else
+      init_homepage
+    end
+  end
+
+  def home
+    if request.cookies['pm'] != 'new_header'
+      redirect_to '/'
+    else
+      init_homepage
+      render 'home/index'
+    end
+  end
+
+  def init_homepage
```

Comment:
Not sure I follow. `init_homepage` is a helper method, not an action, so as we discussed that should be private. Otherwise are you just suggesting that the index route calls `render :index` for symmetry?